src/document/DocumentAnnotator.ts

```diff
@@ -86,17 +86,8 @@ export default class DocumentAnnotator extends BaseAnnotator {
         this.managers.set(pageNumber, pageManagers);
     }
 
-    scrollToAnnotation(annotationId: string | null): void {
-        if (!annotationId) {
-            return;
-        }
-
-        const annotation = getAnnotation(this.store.getState(), annotationId);
-        const annotationPage = annotation?.target.location.value;
-
-        if (!annotation || !annotationPage) {
-            return;
-        }
+    scrollToAnnotation(annotation: Annotation): void {
+        const annotationPage = annotation.target.location.value;
```

Comment:
The naked check here seems very risky to me. What if the annotation passed in isn't in the store? What if its properties are undefined?

Reply:
I was thinking about TS catching that, but using it as a library that won't help. I'll fix it up.